src/diffusers/models/unet_2d.py

```diff
@@ -113,6 +116,14 @@ def __init__(
 
         self.time_embedding = TimestepEmbedding(timestep_input_dim, time_embed_dim)
 
+        # class embedding
+        if class_embed_type == "timestep":
+            self.class_embedding = TimestepEmbedding(timestep_input_dim, time_embed_dim)
+        elif class_embed_type == "identity":
+            self.class_embedding = nn.Identity(time_embed_dim, time_embed_dim)
+        else:
+            self.class_embedding = None
```

Comment:
In `UNet2DConditionModel` if the `class_embed_type` is `None`, we use a learned embeddings matrix.
cf https://github.com/huggingface/diffusers/blob/9dbf78e2f1b04be998ff335a09b7152cce3d719a/src/diffusers/models/unet_2d_condition.py#L144